invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -51,22 +47,13 @@ class Meta:
         'files': 'read_files',
     }
 
-    # schema_version = fields.Interger(dump_only=True)
-    # revision = fields.Integer(attribute='revision_id', dump_only=True)
```

Comment:
Minor: Revision should also be possible to dump directly.

Reply:
Yes, didnt work on commented fields, there might be many more like this.